package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "space-travellers-hub",
+  "version": "0.1.0",
+  "type": "module",
```

Comment:
- [ ] **Optional:**

Kindly remove ` "type": "module",` as this would force you to supply extensions of imported files and result in the errors below that prevent the app from running.

Though this is not required, without it the next reviewer would not be able to review your code easily. I therefore highly recommend it.

![image](https://user-images.githubusercontent.com/27709832/226111353-92c6b874-b19b-4427-a803-5a7520ac7b71.png)